packages/editor/package.json

```diff
@@ -39,14 +39,18 @@
     "apollo-utilities": "^1.3.2",
     "array-move": "^2.2.1",
     "core-js": "^3.4.8",
+    "css-loader": "^4.2.1",
```

Comment:
`css-loader` is a plugin for webpack and only needed during development. It belongs to the `devDependencies`.

Reply:
Ok done